train_segmentation.py

```diff
@@ -545,20 +703,17 @@ def main(params, config_path):
                         'model': state_dict,
                         'best_loss': best_loss,
                         'optimizer': optimizer.state_dict()}, filename)
-            if epoch == 0:
-                log_artifact(filename)
```

Comment:
why were we saving the checkpoing at epoch=0 in mlflow? @mpelchat04 @victorlazio109 ?

Reply:
good question. I don't think it's relevant to do so.